.travis.yml

```diff
@@ -5,6 +5,11 @@ services:
 
```

Comment:
love the detailed instructions in your README. It's given me some great tips on how to improve mine thanks Ara :)